src/noc/containers/noc-submit-success/noc-submit-success.component.html

```diff
@@ -21,8 +21,7 @@ <h2 class="govuk-heading-m ">What happens next</h2>
           <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
           <strong class="govuk-warning-text__text">
             <span class="govuk-warning-text__assistive">Warning</span>
-            You should now amend the case details, for example the unique solicitor reference and the correspondence
-            address.
+            You should now amend the case details, for example the unique solicitor reference, the correspondence address and email address.
```

Comment:
"... the correspondence address, and the email address."

Reply:
the content is from end user I will keep it as it says.

Reply:
Okay, no problem. Thought I'd flag it in case there was a transcribing error. 👍